product-service/src/functions/getProductsById/handler.ts

```diff
@@ -0,0 +1,15 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { middyfy } from '@libs/lambda';
+import products from "@functions/mock-db";
+
+
+const getProductsList: ValidatedEventAPIGatewayProxyEvent<null> = async ({pathParameters: {productId}}) => {
```

Comment:
Probably, it should be renamed to getProductById

Reply:
fixed in pr for Task 4